### DISCUSSION TITLE
When i start turbo in g-helper, I fand my powersetting still is blance

### DISCUSSION THREAD

**Discussion by Du1cckkk** [seerge/g-helper#2007] [Q&A] [answered]
![1706681728285](https://github.com/seerge/g-helper/assets/128781646/686db3f7-b71d-4930-9204-801df35dc0d8)

**Comment by Kikimahe** [comment, marked as answer]
Hi,

It's the 3 Power Modes that are changed, not the power plan. :)

Turbo -> Best Performance
Balanced -> Balanced
Silent -> Best Power efficiency

![Change-Power-Mode-Settings-in-Windows-11-body-2](https://github.com/seerge/g-helper/assets/142881210/bec9355f-8185-4329-acad-4e0ea6dcce7e)

![Capture d'écran 2024-01-31 080827](https://github.com/seerge/g-helper/assets/142881210/5a834463-fecd-41bb-8813-2ec8c72fc24e)

**Reply by Du1cckkk** [reply]
oh! I confused Power Modes and power plan
thanks for your answer, but I fand when i uninstall the Armory Crate,my Turbo power plan has been missed

**Reply by Kikimahe** [reply]
You don't need additional power plans to have the best performance.

Just use the default power plan. :)
